### ISSUE TITLE
Crashes after playing, scrambles controls, can't reboot

### ISSUE THREAD

**Issue by quattj** [loganmc10/m64p#22]
I have Windows 7 64 bit, Biostar AMD 970 motherboard, AMD FX 8570, XFX Radeon RX 460.  Using my Mayflash to Wii Classic adapter.

I am able to play a game, but often after quitting the game and trying to change a setting or load the same and/or another game, the emulator crashes or becomes unresponsive.  My best guess is it is not properly releasing the controllers back to the system when a game ends.  When this happens, if I go in to my windows control panel > game controller settings, my controllers are unresponsive, usually showing two unused axes stuck at 50%.  The emulator may need a failsafe for if controllers are not responding so it also does not lock up.

When it does crash, there is usually an instance of the emulator using 117 MB of RAM that can not be force closed by any method, and about 80% of the time, I will need to hold the power button on my computer to be able to shut down or reboot.

I only started using this emulator earlier this month, so I don't know how long this issue might have been present.  I did just update my video drivers to see if it would help, but it did not.

**Comment by quattj** [comment]
I just experienced this same issue with the most recent monthly release of Dolphin emulator, so I'm thinking there is a problem with my computer that I need to track down, and not necessarily the emulators (unless they share input methods).

**Comment by quattj** [comment]
I'm going to close this issue.  It appears I have a bad USB hub on my motherboard.  The scrambled controllers started happening with more games and more often.  I tried checking in the Windows Device Manager to see if my controllers were drawing too much power or something, and one particular hub crashed the properties window every time I clicked the "power" tab.  After disabling it and moving my controllers to other USB ports, I have not experienced any further issues.
